File: 프로그래머스/0/120830. 양꼬치/양꼬치.java
class Solution {
    public int solution(int n, int k) {
        int answer = 0;
        int service = 0;    // 서비스 음료수 갯수
        
        if(n >= 10) service = n / 10;
        answer = (n * 12000) + (k * 2000) - (service * 2000);   
        
        return answer;
    }
}

File: 프로그래머스/0/120830. 양꼬치/양꼬치.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n, int k) {
    int answer = 0;
    int s = 0; // 음료수 서비스
    
    if(n >= 10) s = n / 10;
    
    answer = n * 12000 + k * 2000 - s * 2000;    
    
    return answer;
}